external-crates/move/crates/move-command-line-common/src/testing.rs

```diff
@@ -63,3 +63,105 @@ pub fn format_diff(expected: impl AsRef<str>, actual: impl AsRef<str>) -> String
     }
     ret
 }
+
+/// See `insta_assert!` for documentation.
+pub struct InstaOptions<Info: serde::Serialize> {
+    pub info: Option<Info>,
+}
+
+impl<Info: serde::Serialize> InstaOptions<Info> {
+    /// See `insta_assert!` for documentation.
+    pub fn new() -> Self {
+        Self { info: None }
+    }
+}
+
+impl InstaOptions<()> {
+    /// See `insta_assert!` for documentation.
+    pub fn none() -> Self {
+        Self { info: None }
+    }
+}
+
+#[macro_export]
+/// A wrapper around `insta::assert_snapshort` to promote uniformity in the Move codebase, intended
+/// to be used with datatest-stable and as a replacement for the hand-rolled baseline tests.
+/// The snapshot file will be saved in the same directory as the input file with the name specified.
+/// In essence, it will be saved at the path `{input_path}/{name}.snap`.
+///
+/// For ease of use and reviewing, `insta_assert` should be used at most once per test. When it
+/// fails, it will stop the test. So if there are multiple snapshots in a given test, it would
+/// require multiple test runs to review all the failures.
```

Comment:
I don't believe this is necessarily true -- see "Disabling Assertion Failure" here: https://insta.rs/docs/advanced/#disabling-assertion-failure. Might be worth mentioning this/pointing to it in case folks do want to generate multiple snapshots per test.

Reply:
I'll mention it, but I think given the fact that you can only skip it via an env (and not something in `Settings`) it is worth advising against. Just a lot of friction and what not